cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/Index.java

```diff
@@ -61,7 +61,7 @@ private Index(List<Object> fieldNames,
      * This method sets the index type to the default setting of "json"
      *
      * @param fieldNames the field names in the index
-     * @param indexName the index name
+     * @param indexName the index name or null
```

Comment:
When I reached the end of the diff I did see that the behaviour is documented on the `ensureIndexed`, which is probably sufficient.